### PR TITLE
Update setup and fix linter issues

### PR DESCRIPTION
**Pull request by cmagnobarbosa**
Update setup and fix linter issues

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -23,6 +23,12 @@ Fixed
 Security
 ========
 
+[2.2.1] - 2019-03-12
+********************
+Fixed
+=====
+- Improvements code organization and fix some linter issues.
```
Comment (hdiogenes) [inline CHANGELOG.rst]: Please, move changelogs to the UNRELEASED section. We can only add a release date when we make a real version release.

Also s/Improvements/Improve/

File: setup.cfg [hunk 2]
```diff
@@ -3,12 +3,16 @@ exclude = .eggs,ENV,build,docs/conf.py,venv
...
 [yala]
 radon mi args = --min C
-pylint args = --disable=too-few-public-methods,too-many-instance-attributes,super-init-not-called
+pylint args = --disable=too-few-public-methods,too-many-instance-attributes,super-init-not-called,too-many-branches
 
 [pydocstyle]
-add-ignore = D105,D107
+add-ignore = D102,D105,D107,D104
 # D105: Missing docstring in magic method
 # D107: Missing docstring in __init__
```
Comment (hdiogenes) [inline setup.cfg:11]: Please follow the same pattern and add a comment line describing what D104 stands for.

File: setup.py [hunk 3]
```diff
@@ -1,21 +1,49 @@
...
+# Kytos var folder
+VAR_PATH = BASE_ENV / 'var' / 'lib' / 'kytos'
+# Path for enabled NApps
+ENABLED_PATH = VAR_PATH / 'napps'
+# Path to install NApps
+INSTALLED_PATH = VAR_PATH / 'napps' / '.installed'
+CURRENT_DIR = Path('.').resolve()
+
+# NApps enabled by default
+CORE_NAPPS = ['of_core']
```
Comment (hdiogenes) [inline setup.py]: Pathfinder install doesn't depend on `of_core`, so you can just leave CORE_NAPPS empty.

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (cmagnobarbosa) [reply CHANGELOG.rst]: Thanks for review. I updated the changelog.

Comment (cmagnobarbosa) [reply CHANGELOG.rst]: Thanks for review. Solved in last push.

File: setup.cfg (hunk 2 above)
Comment (cmagnobarbosa) [reply setup.cfg:11]: Thanks. Solved

Comment (cmagnobarbosa) [reply setup.cfg:11]: Thanks. Solved.

File: setup.py (hunk 3 above)
Comment (cmagnobarbosa) [reply setup.py]: Updated. Solved in last push.

**Pull request merged**